opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseDrawerInteractor.java

```diff
@@ -65,4 +75,55 @@ public void run() {
         appExecutors.diskIO().execute(runnable);
 
     }
+
+    @Override
+    public void checkSynced() {
+
+        String syncQuery = "SELECT syncStatus FROM client WHERE syncStatus <> 'Synced'\n" +
+                "UNION ALL\n" +
+                "SELECT syncStatus FROM event WHERE syncStatus <> 'Synced' AND syncStatus <> 'task_unprocessed' \n" +
+                "UNION ALL\n" +
+                "SELECT sync_Status FROM task WHERE sync_Status <> 'Synced'\n" +
+                "UNION ALL\n" +
+                "SELECT sync_Status FROM structure WHERE sync_Status <> 'Synced'\n";
+
+        Runnable runnable = new Runnable() {
+            @Override
+            public void run() {
+                Cursor syncCursor = null;
+                try
+                {
+                    syncCursor  = database.rawQuery(syncQuery, null);
+                    Integer count = syncCursor.getCount();
+
+                    if(count == 0)
+                    {
+                        RevealApplication.getInstance().setSynced(true);
+                    }
+                    else
+                    {
+                        RevealApplication.getInstance().setSynced(false);
```

Comment:
Use the variable declared [here](https://github.com/OpenSRP/opensrp-client-reveal/pull/746/files#diff-d5fca77e8ee15c2bb1d3772213a1a19eR29)